lib/data/multipleBackendGateway.js

```diff
@@ -62,7 +63,28 @@ const multipleBackendGateway = {
         return client.delete(objectGetInfo, reqUids, callback);
     },
 
-    // checkHealth:
+    healthcheck: (log, callback) => {
+        const multBackendResp = {};
+        async.each(Object.keys(clients), (location, cb) => {
+            const client = clients[location];
+            if (client.clientType && client.clientType === 'scality') {
```

Comment:
The client.clientType check is there because not every backend gets assigned a clientType, so it would be undefined. And client should never be undefined because that would mean the locationConfig.json was empty, which will throw an error on startup :)